2024-03-08 11:39:16,016 INFO    MainThread:14416 [wandb_setup.py:_flush():76] Configure stats pid to 14416
2024-03-08 11:39:16,016 INFO    MainThread:14416 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hugoa\.config\wandb\settings
2024-03-08 11:39:16,016 INFO    MainThread:14416 [wandb_setup.py:_flush():76] Loading settings from c:\Users\hugoa\H9\RL\GitHub Vrain\rl_levitation_control\rl_levitation_control\wandb\settings
2024-03-08 11:39:16,016 INFO    MainThread:14416 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-08 11:39:16,016 INFO    MainThread:14416 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-08 11:39:16,016 INFO    MainThread:14416 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-08 11:39:16,017 INFO    MainThread:14416 [wandb_init.py:_log_setup():506] Logging user logs to c:\Users\hugoa\H9\RL\GitHub Vrain\rl_levitation_control\rl_levitation_control\wandb\run-20240308_113916-nyk3tq5f\logs\debug.log
2024-03-08 11:39:16,017 INFO    MainThread:14416 [wandb_init.py:_log_setup():507] Logging internal logs to c:\Users\hugoa\H9\RL\GitHub Vrain\rl_levitation_control\rl_levitation_control\wandb\run-20240308_113916-nyk3tq5f\logs\debug-internal.log
2024-03-08 11:39:16,017 INFO    MainThread:14416 [wandb_init.py:init():546] calling init triggers
2024-03-08 11:39:16,017 INFO    MainThread:14416 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {}
2024-03-08 11:39:16,017 INFO    MainThread:14416 [wandb_init.py:init():568] re-initializing run, found existing run on stack: l7ktcv3k
2024-03-08 11:39:16,019 INFO    MainThread:14416 [wandb_run.py:_finish():1854] finishing run hyperloopupv/2gdl/l7ktcv3k
2024-03-08 11:39:16,020 ERROR   MainThread:14416 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-03-08 11:39:16,020 INFO    MainThread:14416 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-08 11:39:16,020 INFO    MainThread:14416 [wandb_init.py:_jupyter_teardown():434] cleaning up jupyter logic
2024-03-08 11:39:16,021 INFO    MainThread:14416 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2024-03-08 11:39:16,021 INFO    MainThread:14416 [wandb_run.py:_restore():2066] restore
2024-03-08 11:39:16,021 INFO    MainThread:14416 [wandb_run.py:_restore():2072] restore done
2024-03-08 11:39:19,091 INFO    MainThread:14416 [wandb_run.py:_footer_history_summary_info():3422] rendering history
2024-03-08 11:39:19,091 INFO    MainThread:14416 [wandb_run.py:_footer_history_summary_info():3454] rendering summary
2024-03-08 11:39:19,094 INFO    MainThread:14416 [wandb_run.py:_footer_sync_info():3380] logging synced files
2024-03-08 11:39:19,098 INFO    MainThread:14416 [wandb_init.py:init():602] starting backend
2024-03-08 11:39:19,098 INFO    MainThread:14416 [wandb_init.py:init():606] setting up manager
2024-03-08 11:39:19,101 INFO    MainThread:14416 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2024-03-08 11:39:19,103 INFO    MainThread:14416 [wandb_init.py:init():613] backend started and connected
2024-03-08 11:39:19,104 INFO    MainThread:14416 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2024-03-08 11:39:19,104 INFO    MainThread:14416 [wandb_run.py:_label_probe_notebook():1214] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-03-08 11:39:19,104 INFO    MainThread:14416 [wandb_init.py:init():701] updated telemetry
2024-03-08 11:39:19,257 INFO    MainThread:14416 [wandb_init.py:init():741] communicating run to backend with 60.0 second timeout
2024-03-08 11:39:20,595 INFO    MainThread:14416 [wandb_run.py:_on_init():2133] communicating current version
2024-03-08 11:39:20,660 INFO    MainThread:14416 [wandb_run.py:_on_init():2142] got version response upgrade_message: "wandb version 0.16.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-03-08 11:39:20,660 INFO    MainThread:14416 [wandb_init.py:init():789] starting run threads in backend
2024-03-08 11:39:21,030 INFO    MainThread:14416 [wandb_run.py:_console_start():2114] atexit reg
2024-03-08 11:39:21,030 INFO    MainThread:14416 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2024-03-08 11:39:21,030 INFO    MainThread:14416 [wandb_run.py:_redirect():2034] Wrapping output streams.
2024-03-08 11:39:21,030 INFO    MainThread:14416 [wandb_run.py:_redirect():2059] Redirects installed.
2024-03-08 11:39:21,030 INFO    MainThread:14416 [wandb_init.py:init():831] run started, returning control to user process
2024-03-08 11:46:07,703 INFO    MainThread:14416 [wandb_setup.py:_flush():76] Configure stats pid to 14416
2024-03-08 11:46:07,703 INFO    MainThread:14416 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hugoa\.config\wandb\settings
2024-03-08 11:46:07,703 INFO    MainThread:14416 [wandb_setup.py:_flush():76] Loading settings from c:\Users\hugoa\H9\RL\GitHub Vrain\rl_levitation_control\rl_levitation_control\wandb\settings
2024-03-08 11:46:07,703 INFO    MainThread:14416 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-08 11:46:07,703 INFO    MainThread:14416 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-08 11:46:07,703 INFO    MainThread:14416 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-08 11:46:07,703 INFO    MainThread:14416 [wandb_init.py:_log_setup():506] Logging user logs to c:\Users\hugoa\H9\RL\GitHub Vrain\rl_levitation_control\rl_levitation_control\wandb\run-20240308_114607-6ks22kax\logs\debug.log
2024-03-08 11:46:07,704 INFO    MainThread:14416 [wandb_init.py:_log_setup():507] Logging internal logs to c:\Users\hugoa\H9\RL\GitHub Vrain\rl_levitation_control\rl_levitation_control\wandb\run-20240308_114607-6ks22kax\logs\debug-internal.log
2024-03-08 11:46:07,704 INFO    MainThread:14416 [wandb_init.py:_jupyter_setup():452] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001B04A35F610>
2024-03-08 11:46:07,704 INFO    MainThread:14416 [wandb_init.py:init():546] calling init triggers
2024-03-08 11:46:07,704 INFO    MainThread:14416 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {}
2024-03-08 11:46:07,704 INFO    MainThread:14416 [wandb_init.py:init():568] re-initializing run, found existing run on stack: nyk3tq5f
2024-03-08 11:46:07,706 INFO    MainThread:14416 [wandb_run.py:_finish():1854] finishing run hyperloopupv/5gdl/nyk3tq5f
2024-03-08 11:46:07,706 INFO    MainThread:14416 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2024-03-08 11:46:07,706 INFO    MainThread:14416 [wandb_run.py:_restore():2066] restore
2024-03-08 11:46:07,706 INFO    MainThread:14416 [wandb_run.py:_restore():2072] restore done
2024-03-08 11:46:12,347 INFO    MainThread:14416 [wandb_run.py:_footer_history_summary_info():3422] rendering history
2024-03-08 11:46:12,347 INFO    MainThread:14416 [wandb_run.py:_footer_history_summary_info():3454] rendering summary
2024-03-08 11:46:12,351 INFO    MainThread:14416 [wandb_run.py:_footer_sync_info():3380] logging synced files
